src/redux/missions/missionsSlice.js

```diff
@@ -0,0 +1,70 @@
+/* eslint no-param-reassign: ["error", { "props": false }] */
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+// import axios from 'axios';
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure you are not disabling some linters rules since they help your code to suit the standards of clean code.
- [ ] Also consider deleting the commented lines in your codebase